ddtrace/contrib/psycopg/patch.py

```diff
@@ -61,12 +61,16 @@ def unpatch():
 class Psycopg2TracedCursor(dbapi.TracedCursor):
     """TracedCursor for psycopg2"""
 
-    def _trace_method(self, method, name, resource, extra_tags, *args, **kwargs):
+    def _trace_method(self, method, name, resource, extra_tags, dbm_operation, *args, **kwargs):
         # treat psycopg2.sql.Composable resource objects as strings
         if isinstance(resource, Composable):
             resource = resource.as_string(self.__wrapped__)
+            # DBM context propagation is only supported for query args with the following type: string
+            dbm_operation = False
```

Comment:
Does a test fail if this is not disabled?

Reply:
Yup a few tests will fail. Here's an example: https://github.com/DataDog/dd-trace-py/blob/1.x/tests/contrib/psycopg/test_psycopg.py#L316

Reply:
Ok, we will want to revisit this as part of a clean up of the psycopg integration.